app/views/cohorts/columns/edit.haml

```diff
@@ -31,7 +31,10 @@
             - unless column.input_type == 'read_only'
               %input{type: 'checkbox', value: '1', checked: editable, name: "column_state[editable][#{column.column}]", id: "column_state_editable_#{column.column}"}
           %td
-            = column.title
+            - if can_edit_translations? && column.attributes.include?(:translation_key)
```

Comment:
I like how easy this is, but... I think it's too easy.  It's not clear that when I adjust the title of one column, say "Rank" that I'm actually changing all cohorts.  Also, there's no indication of what it was before it was translated, so going back is difficult.    Also this modal is all about re-ordering.  

I propose, we create a page linked from `https://boston-hmis.dev/cohorts` along side the  Edit Cohort Column Options (maybe even nested within that).  It should only allow translation of columns that are editable (don't inherit from ReadOnly -- those should have to be translated in the translation interface to avoid mass confusion.)

The page should have:
* Untranslated string
* Translation field
* column type, string, date, boolean, etc. (if it's not too hard to obtain.)